tests/Feature/Booking/BookingTest.php

```diff
@@ -260,6 +277,89 @@ public function testInvalidBookingCantBeCreated()
         );
     }
 
+    /** @test */
+    public function testOverlappingBookings()
```

Comment:
Can you split this test up?

Reply:
Not sure it would be beneficial, would just have to create the test bookings & positions every time